tools/font-edit/README.md

```diff
@@ -14,4 +14,13 @@ make
 ./twin-fedit < nchars
 ```
 
-(press 'q' to next character)
+| Key binding | Function |
+|   ---       |   ---    |
+| `esc`       | Exit program |
```

Comment:
Don't use [backtick](https://en.wikipedia.org/wiki/Backtick) characters within this table. All keys here are common.